docs/libraries/how-to-build-a-library.md

```diff
@@ -4,6 +4,13 @@
 
 - JDK 8
 
+For [Arrow Fx](https://github.com/arrow-kt/arrow-fx):
+
+- Android SDK
+- `ANDROID_SDK_ROOT` environment variable for 
```

Comment:
for? 😄

Reply:
Thanks for the review @aballano , fixed!!